.pre-commit-config.yaml

```diff
@@ -34,18 +34,22 @@ repos:
 # TERRAFORM #
 #############
 - repo: git://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.28.0
+  rev: v1.43.0
   hooks:
     - id: terraform_fmt
     - id: terraform_docs
-    - id: terraform_validate
+
+- repo: https://github.com/gruntwork-io/pre-commit
+  rev: v0.1.10
+  hooks:
+    - id: terraform-validate
```

Comment:
Babenko `terraform_validate` hook not work at all?

Reply:
It requires additional setup (running terraform init for every folder) vs gruntwork hook just works.